<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-R8BAB90" failures="2" tests="2" name="Login Test" time="87.206" errors="0" timestamp="15 Apr 2020 12:29:28 GMT">
  <testcase classname="sample.recovery.samplerecovery.RediffLogin" name="RediffLogintest" time="45.025">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Object Not found : pwd_tb_xpath">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	Object Not found : pwd_tb_xpath
	at sample.recovery.samplerecovery.keywords.GenericKeywords.assertall(GenericKeywords.java:521)
	at sample.recovery.samplerecovery.driver.DriverScript.executeKeywords(DriverScript.java:103)
	at sample.recovery.samplerecovery.RediffLogin.RediffLogintest(RediffLogin.java:33)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- RediffLogintest -->
  <testcase classname="sample.recovery.samplerecovery.RediffLogin" name="RediffLogintest" time="39.469">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Object Not found : pwd_tb_xpath">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	Object Not found : pwd_tb_xpath
	at sample.recovery.samplerecovery.keywords.GenericKeywords.assertall(GenericKeywords.java:521)
	at sample.recovery.samplerecovery.driver.DriverScript.executeKeywords(DriverScript.java:103)
	at sample.recovery.samplerecovery.RediffLogin.RediffLogintest(RediffLogin.java:33)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- RediffLogintest -->
</testsuite> <!-- Login Test -->
